src/SFA.DAS.Commitments.Api.IntegrationTests/Tests/WhenSimulatingRealWorldApprenticeshipLoad.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MoreLinq;
+using NUnit.Framework;
+using SFA.DAS.Commitments.Api.Controllers;
+using SFA.DAS.Commitments.Api.DependencyResolution;
+using SFA.DAS.Commitments.Api.IntegrationTests.ApiHost;
+using SFA.DAS.Commitments.Api.IntegrationTests.DatabaseSetup;
+using SFA.DAS.Commitments.Api.IntegrationTests.Helpers;
+
+namespace SFA.DAS.Commitments.Api.IntegrationTests.Tests
+{
+    [TestFixture]
+    public class WhenSimulatingRealWorldApprenticeshipLoad
+    {
+        //todo: better handling of testids
+        [Test]
+        public async Task SimulateSlowdownScenario()
+        {
+            const int numberOfGetApprenticeshipTasks = 30, getApprenticeshipCallsPerTask = 6,
+                numberOfGetApprenticeshipsTasks = 1, getApprenticeshipsCallsPerTask = 3;
+
+            await TestLog.Progress("Starting SimulateSlowdownScenario() Test");
+            await TestLog.Progress("Generating call parameters");
+
+            var getApprenticeshipCallParamsPerTaskTask = GetGetApprenticeshipCallParamsPerTask(numberOfGetApprenticeshipTasks, getApprenticeshipCallsPerTask);
```

Comment:
variable naming is slightly confusing. GetGetApprenticeshipCallParamsPerTask and getApprenticeshipCallParamsPerTaskTask are slightly ambiguous.

Reply:
Renamed GetXxxCallParamsPerTask -> GenerateXxxCallParamsPerTask, which is much clearer.
The XxxTaskTask name looks odd, but it is descriptive, it's a Task to get the CallParamsPerTask.